src/ResourceManagement/ServiceFabric/Microsoft.Azure.Management.ServiceFabric/generate.cmd

```diff
@@ -0,0 +1,17 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+set autoRestVersion=1.0.0-Nightly20170202
+if  "%1" == "" (
+    set specFile="https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-servicefabric/2016-09-01/swagger/servicefabric.json"
```

Comment:
There is another PR is pending for review for the Swagger file, but the review should be a quick review mainly forces on description part., i will point it to the latest commit after it is done.